CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ Per Keep a Changelog there are 6 main categories of changes:
 
 ## Unreleased
 
+- `Reader::levels` now returns an iterator over `Level` structs, which contain the bytes of the level as well as the uncompressed length (by @expenses).
+- Internal: Some previously internal functions were made public, such as `LevelIndex::from_bytes` (by @expenses).
```

Comment:
Introducing new public APIs is not an internal change. We should furthermore enumerate the new APIs here for discoverability.